### ISSUE TITLE
sketcher preferences drop down items

### ISSUE THREAD

**Issue by kinghat** [obelisk79/OpenDark#59]
some of the items in preferences -> sketcher -> grid are hard to read:

![Screenshot_20240126_213653](https://github.com/obelisk79/OpenDark/assets/16073360/c6dcf8d0-e459-41d9-b615-661d96835a28)

**Comment by obelisk79** [comment]
Are you referring to the line patterns? I also notice the checkbox markers appear wonky, but I don't see that on my machine. Windows/Mac?

**Comment by kinghat** [comment]
> Are you referring to the line patterns? I also notice the checkbox markers appear wonky, but I don't see that on my machine.

yes the line patterns in the drop down. i should have been more clear.

> Windows/Mac?

```
OS: Fedora Linux 39 (KDE Plasma) (KDE/plasma)
Word size of Ondsel: 64-bit
Version: 2024.1.0.35694 (Git) AppImage
Build type: Release
Branch: flavor
Hash: b070090c6873b3fcf4be9a459219d22edfc4e7cf
Python 3.10.13, Qt 5.15.8, Coin 4.0.0, Vtk 9.2.6, OCC 7.6.3
Locale: English/United States (en_US)
Installed mods: 
  * OpenDark 2023.12.17
  * Ondsel-Lens 2024.1.22.01
```

**Comment by maxwxyz** [comment]
Still present in OpenTheme.

No issue in OpenLight obviously:
![image](https://github.com/obelisk79/OpenDark/assets/6246609/3c6ee247-78c6-40c9-a3bc-7f03823ff453)
